test/lib/test-init.ts

```diff
@@ -0,0 +1,42 @@
+import { fork } from 'child_process';
+import { exit } from 'process';
+export const testListenPort = 1337;
+export const testLocalServer = `http://localhost:${testListenPort}`;
+
+export async function testInit(
+	fixturePath: string,
+	deleteDb: boolean = false,
+	pineListenPort: number = testListenPort,
+) {
+	try {
+		const processArgs = {
+			fixturePath,
+			testListenPort: pineListenPort,
+			deleteDb: deleteDb || process.env.DELETE_DB,
+		};
+
+		const testServer = fork(
+			__dirname + '/pine-in-process.ts',
+			[JSON.stringify(processArgs)],
+			{ detached: false, execArgv: ['-r', 'ts-node/register'] },
+		);
+		await new Promise((resolve, reject) => {
+			testServer.on('message', (msg: any) => {
```

Comment:
Doesn't it have a default type that's fairly relaxed already?
```suggestion
			testServer.on('message', (msg) => {
```

Reply:
Typed as `{ init: string }` from the Chilprocess' process.send({ init: 'success' }). Thx.